packages/experiments/src/components/Stack/Stack.types.ts

```diff
@@ -73,7 +73,7 @@ export interface IStackProps
    * Defines whether Stack children should not shrink to fit the available space.
    * @defaultvalue false
    */
-  preventShrink?: boolean;
+  disableShrink?: boolean;
```

Comment:
I almost feel like the naming is inverted.. it seems like it should be `disableChildShrink` here and `disableShrink` in `StackItem`.

Reply:
Hmmm, I can see where you're coming from, so that `disableChildShrink` applies to the children and `disableShrink` applies to the item itself. I don't have a problem in changing it, however, if that's the case, wouldn't `disableChildrenShrink` be better than `disableChildShrink`.

Reply:
Maybe `disableShrink` is fine in both cases since the essence of Stack is its children....

Reply:
That would make sense to me too. Actually I prefer that, makes it easier from an API standpoint.

Reply:
Changed 👍